### ISSUE TITLE
Line by line comparison of output with Matlab code

### ISSUE THREAD

**Issue by iandanforth** [iandanforth/pymuscle#26]
After attempting #22 and #23 I discovered the model produces unexpected results. Fatigue is far too fast. I spent several hours trying to debug my code implementing the equations from the paper to no avail. 

So now I need to go back to [the original code](https://github.com/iandanforth/Potvin_Fuglevand_Muscle_Fatigue_Model) and compare outputs at each stage very carefully to detect bugs.

**Comment by iandanforth** [comment]
Normalized Forces - CHECKED

The forces appear identical in the sigmoid portion but the linear portion
differss. The code here differs from the paper.

**Comment by iandanforth** [comment]
The code I had for the nominal fatigue rates was pretty far off from the matlab code. I'm not sure how to go from equation (9) to the matlab code.  

**Code**
```matlab
%% Calculation of Fatigue Parameters (recovery currently set to zero in this version)
    
% note, if rp = 100 & fatigue_range = 180, there will be a 100 x 180 = 1800-fold difference in
% the absolute fatigue of the highest threshold vs the lowest threshold.
% The highest threshold MU will only achieve ~57% of its maximum (at 25 Hz), so the actual range of fatigue
% rates is 1800 x 0.57 = 1026 

% fatigue rate for each motor unit  (note: "log" means "ln" in Matlab)
b2 = log(fatigue_range)/(nu-1);       
mufatrate = exp(b2 * (n-1)); % Curve from 1 to 180

% NOMINAL FATIGUE RATES
fatigue = zeros(1,nu); % VERIFIED
for mu = 1:nu
     fatigue(mu) = mufatrate(mu) * (FatFac / fat) * P(mu);    
        % the full fatigue rate is mufatrate(mu) * [max_fatigue_rate / fatigue_range] * Pr(mu,act) * P(mu)
        % the only variable is the relative force: Pr(mu,act), so this part is calculated once here
end
```

**Paper**
![image](https://user-images.githubusercontent.com/446062/42247998-419831ae-7ed8-11e8-8bda-62df29de4540.png)

**Comment by iandanforth** [comment]
After changing the above my version of chart 1.D doesn't look quite right. The curve in 1.D is pretty flat whereas mine has a strong bow.

![image](https://user-images.githubusercontent.com/446062/42248270-8f8fc70e-7ed9-11e8-9c5a-43172f52229a.png)

![image](https://user-images.githubusercontent.com/446062/42248285-a3c36564-7ed9-11e8-8538-9a547d8b3d0a.png)

**Comment by iandanforth** [comment]
I have now [vectorized much of the original matlab code](https://github.com/iandanforth/Potvin_Fuglevand_Muscle_Fatigue_Model/blob/master/revised.m). This let me quickly iterate on debugging my code. I found several small errors, the largest of which were around adapting firing rates and contraction times. Those have been resolved. 

The charts output from the target force simulations now closely (almost exactly) match the figures from the paper.
